source/common/upstream/outlier_detection_impl.cc

```diff
@@ -222,12 +222,22 @@ DetectorConfig::DetectorConfig(const envoy::api::v2::cluster::OutlierDetection&
           PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, success_rate_request_volume, 100))),
       success_rate_stdev_factor_(static_cast<uint64_t>(
           PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, success_rate_stdev_factor, 1900))),
+      failure_percentage_threshold_(static_cast<uint64_t>(
+          PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, failure_percentage_threshold, 85))),
```

Comment:
I know this is an established pattern in this file. But it would be good to replace these magic numbers with named constants. Similar to how it is done for Http2Settings here: https://github.com/envoyproxy/envoy/blob/d504fde0ffd97017d1ddff8caa9a3b46bba9ae48/include/envoy/http/codec.h#L230

Reply:
Would it be worth doing in this PR, or a followup?

Reply:
Followup PR is fine.

Reply:
+1, thank you